packages/react-core/src/components/ChipGroup/Chip.tsx

```diff
@@ -91,6 +91,7 @@ class Chip extends React.Component<ChipProps & InjectedOuiaProps, ChipState> {
         <Tooltip position={tooltipPosition} content={children}>
           <Component
             className={css(styles.chip, isReadOnly && styles.modifiers.readOnly, className)}
+            tabIndex="0"
```

Comment:
why this?

Reply:
This change comes from merging `master` into `v4`.